docs/src/markdown/faq.md

```diff
@@ -0,0 +1,57 @@
+# Frequently Asked Questions
+
+## Hex Uppercase
+
+> How do I output hex in uppercase?
+
+Some people prefer hex to all be in uppercase, and some may prefer lowercase. ColorHelper, by default, outputs strings
+in lowercase. But worry not, as you can change this via the settings. ColorHelper 2 used to have a `upper_case_hex`
+option, but ColorHelper 3 is a bit different.
+
+In order to make ColorHelper more flexible than ever, we created a dependency called [`coloraide`][coloraide]
+which is used to handle all the conversions, CSS string parsing, and CSS string output. This library allows for each
+color space to define it's string output [options][coloraide-strings]. These options are exposed via in the settings.
```

Comment:
In line 13, "it's" should be "its", and "via in the settings" should just be "via the settings" (or "in the settings", your choice. Just not both...)

Reply:
Thanks! I rushed putting it up as it explained things better. Didn't have a chance to proofread yet 🙂.